src/ui/display_write_video_thread.py

```diff
@@ -13,8 +13,9 @@
 class Thread1(QThread):
     ImageUpdate = pyqtSignal(QImage)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__()
+    def __init__(self, cameraNumber: int, parent = None):
+        super(Thread1, self).__init__(parent)
+        self.cameraNumber = cameraNumber
         self.ThreadActive = True #- see if commenting this out works
```

Comment:
This is optional, but Python tends to use snake case for variables (camera_number), CapitalCase for Classes (VideoThread), and avoids camel case entirely (cameraNumber). This doesn't affect us at all, but does make harder. Designer is stupid and uses camel because Qt is a Cython package. 

You do NOT need to change anything about this. This is just a notice.
```suggestion
        self.thread_active = True #- see if commenting this out works
```